ocs_sample_library_preview/OCSClient.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, api_version: str, tenant: str, url: str, client_id: str,
         self.__subscriptions = Subscriptions(self.__base_client)
         self.__topics = Topics(self.__base_client)
         self.__users = Users(self.__base_client)
+        self.__sharedStreams = SharedStreams(self.__base_client)
```

Comment:
I wouldn't hate alphabetizing this list too, but this new line isn't the only one out of order :)

Reply:
Alphabetized them all :)